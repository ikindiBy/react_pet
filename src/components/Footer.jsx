import React, { Component } from "react";
import "./Footer.scss";

class Footer extends Component {
  render() {
    return (
      <footer>
        <div className="footer-wrapper ">
          <div className="logo-line">logo</div>
          <p>Aliaksandr Filitaryn, Epam, 2019</p>
        </div>{" "}
      </footer>
    );
  }
}

export default Footer;
