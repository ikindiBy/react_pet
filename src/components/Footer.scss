@import "./../styles/utils/vars.scss";
@import "./../styles/utils/mixins.scss";

footer {
  @include size(100%, 70px);
  background-color: $mainAccentColor;
  margin: 0 auto;
  text-align: center;
}

.footer-wrapper {
  width: 80%;
  margin: 0 auto;
}
