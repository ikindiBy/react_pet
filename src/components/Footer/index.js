import React from "react";
import "./Footer.scss";

const Footer = ({ year }) => {
  return (
    <footer>
      <div className="footer-wrapper ">
        <div className="logo-line">logo</div>
        <p>Aliaksandr Filitaryn, Epam, {year}</p>
      </div>
    </footer>
  );
};

export default Footer;
