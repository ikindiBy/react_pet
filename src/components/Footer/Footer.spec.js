import React from "react";
import { shallow, mount } from "enzyme";
import { Footer } from "./Footer.jsx";
// import Adapter from "enzyme-adapter-react-16";

describe("Footer component", () => {
  let year = "2019";

  beforeAll(() => {
    // Enzyme.configure({ adapter: new Adapter() });
    //   Date.now = jest.fn(() => Date.UTC(2017, 0, 1, 1, 1, 1, 1));

    //     todo = {
    //         title: 'test',
    //         id: 1,
    //         complited: false,
    //         created: Date.now()
    //     }
    year = "2019";
  });

  it("should be render correctly", () => {
    const component = shallow(<Footer year={year} />);
    expect(component).toMatchSnapshot();
  });

  // it('should be call onRemove', () => {
  //     const onRemove = jest.fn();
  //     const component = mount(<Item todo={todo} onRemove={onRemove}/>);
  //     const btn = component.find('.btn').simulate('click');
  //     expect(onRemove).toHaveBeenCalled();
  // });
});
