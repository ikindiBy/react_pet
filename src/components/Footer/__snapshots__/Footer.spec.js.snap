// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should be render correctly 1`] = `
<footer>
  <div
    className="footer-wrapper "
  >
    <div
      className="logo-line"
    >
      logo
    </div>
    <p>
      Aliaksandr Filitaryn, Epam, 
      2019
    </p>
  </div>
</footer>
`;
